packages/zstandard/zstandard.v0.12.0/opam

```diff
@@ -13,7 +13,7 @@ depends: [
   "ocaml"       {>= "4.07.0"}
   "core_kernel" {>= "v0.12" & < "v0.13"}
   "ppx_jane"    {>= "v0.12" & < "v0.13"}
-  "conf-zstd"
+  "conf-zstd-1.3.8"
```

Comment:
This should be `conf-zstd {=“1.3.8”}`

Reply:
Indeed; that is an embarrassing mistake...
Thanks!